### PR TITLE
ci: add format check

### PR DESCRIPTION
**Pull request by takashimamorino**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -32,6 +32,9 @@ jobs:
       - name: Build
         run: yarn build
 
+      - name: Format Check
```
Comment (takashimamorino) [inline .github/workflows/ci.yml:35]: This file is the main one.

File: packages/components/src/components/FooterExtended.styles.tsx [hunk 2]
```diff
@@ -9,7 +9,7 @@ export const Container = styled.div(() => [tw`container-max`]);
 export const Row = styled.div(
   ({ equalPadding }: { equalPadding?: boolean }) => [
     tw`flex flex-wrap lg:flex-nowrap`,
-    equalPadding ? tw`py-4 md:(py-8)` : tw`pt-2 pb-4 md:(pt-4 pb-8)`
+    equalPadding ? tw`py-4 md:(py-8)` : tw`pt-2 pb-4 md:(pt-4 pb-8)`,
```
Comment (takashimamorino) [inline packages/components/src/components/FooterExtended.styles.tsx:12]: yarn format

File: packages/components/src/components/HeroGradient.tsx [hunk 3]
```diff
@@ -49,10 +49,12 @@ export const HeroGradient: React.FC<IHeroGradientProps> = ({
             <span {...restProps.versionProps}>{version}</span>
           )}
         </CTA>
-        {image && (
-          React.isValidElement(image) ?
-            image : <Image src={image.src} alt={image.alt} {...restProps.imageProps} />
-        )}
+        {image &&
```
Comment (takashimamorino) [inline packages/components/src/components/HeroGradient.tsx:52]: yarn format

File: packages/components/src/components/SchemaType.tsx [hunk 4]
```diff
@@ -17,7 +17,10 @@ import {
   Frameworks,
 } from './SchemaTypes.styles';
 import { Tag, TagsContainer } from './Tag';
-import { SchemaEditor, ExecutableDocumentEditor } from '../../../monaco-graphql-editor/src';
+import {
```
Comment (takashimamorino) [inline packages/components/src/components/SchemaType.tsx:20]: yarn format

File: packages/components/src/components/SearchBar.tsx [hunk 5]
```diff
@@ -269,13 +269,15 @@ const Hits: React.FC<{ hits: Hit<any>[]; accentColor: string }> = ({
               );
             }
 
-            const isSameWebsite = typeof window === 'object' && subHit.url.startsWith(window.location.origin);
+            const isSameWebsite =
```
Comment (takashimamorino) [inline packages/components/src/components/SearchBar.tsx:272]: yarn format

File: packages/components/src/helpers/assets.tsx [hunk 6]
```diff
@@ -120,8 +120,7 @@ export const PRODUCTS: {
   },
   {
     children: 'Shield',
-    title:
-      'GraphQL Permissions Framework For Complex Authorisation Systems',
+    title: 'GraphQL Permissions Framework For Complex Authorisation Systems',
```
Comment (takashimamorino) [inline packages/components/src/helpers/assets.tsx:123]: yarn format

File: packages/components/src/types/components.ts [hunk 7]
```diff
@@ -328,7 +328,7 @@ export interface IEditorProps {
   schema?: string;
   icon: string;
   image?: string;
-  operations?: string
+  operations?: string;
```
Comment (takashimamorino) [inline packages/components/src/types/components.ts:331]: yarn format

File: packages/monaco-graphql-editor/src/editor/ExecutableDocumentEditor.tsx [hunk 8]
```diff
@@ -164,7 +164,7 @@ export const ExecutableDocumentEditor: FC<
           model: monaco.editor.IReadOnlyModel,
           position: monaco.Position
         ): monaco.languages.CompletionList {
-          const isUriEquals = model.uri.path === editorUriRef.current!.path
+          const isUriEquals = model.uri.path === editorUriRef.current!.path;
```
Comment (takashimamorino) [inline packages/monaco-graphql-editor/src/editor/ExecutableDocumentEditor.tsx:167]: yarn format

**Review by saihaj — APPROVED** [review]
awesome 🚀

**Pull request merged**